### ISSUE TITLE
can't claim grants on staging when using `aab` builds

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#15692]
## Description 

Originally found/reported by @stephendonner, it looks like we can't claim grants on staging due to the following:

```
2021-05-16 19:22:11.331 31751-31751/? V/chromium: [<VERBOSE6:logging_util.cc>(136)] 
    [ RESPONSE - OnRequest ]
    > Url: https://grant.rewards.bravesoftware.com/v2/attestations/safetynet/90d8d547-cb56-4fd0-a31a-5f87800cacff
    > Result: Failure
    > HTTP Code: 400
    > Body: {"message":"Error verifying token: No matching APK signing certificate found","code":400}
```

We should also check and ensure this isn't happening on production or we'll run into a lot support/issues when users start receiving their ad grants.

## Steps to reproduce 

   1. install Brave using an internal sharing link (used `1.24.83 Chromium: 90.0.4430.93`)
   2. change rewards to `staging` using the developer settings
   3. once switched to `staging`, enable rewards and attempt to claim a grant

## Actual result 

<p align="center">
<img src="https://user-images.githubusercontent.com/2602313/117517879-80ab3f00-af6b-11eb-9add-9aaa616303aa.jpg" width=300>
</p>

## Expected result

Should be able to claim/use grants on `staging`.

## Issue reproduces how often 

100% reproducible using the above STR.

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? `N/A` (Yes when using `1.24.x RC`)
- Can you reproduce this issue with the current Play Store Beta version? `Yes`
- Can you reproduce this issue with the current Play Store Nightly version? `Yes`

## Device details

   - Install type (ARM, x86): `ARM`
   - Device type (Phone, Tablet, Phablet): `Samsung S10+`
   - Android version: `Android 11`
   
## Brave version

* `1.24.83 Chromium: 90.0.4430.93` - **Reproduced**

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? `N/A`
- Does the issue resolve itself when disabling Brave Rewards? `N/A`
- Is the issue reproducible on the latest version of Chrome? `N/A`

### Additional information

CCing @SergeyZhukovsky @anthonypkeane @bsclifton @brave/legacy_qa @evq

**Comment by kjozwiak** [comment]
Labeled as `release/blocking` as we need to make sure this isn't happening on production either.

**Comment by kristoferbartinik** [comment]
I have the same problem.
Brave: 1.23.73 Chromium:90.0.4430.93
I made a post in the community, however, it fell into hold, saying it needs moderation, so far nothing and I can't even comment in the community.

**Comment by kjozwiak** [comment]
Removing the `release/blocking` as this works when claiming grants on `aab` builds that were uploaded into GPS. We'll need to figure out how to get around this re: being able to claim grants on staging on `aab` builds that were downloaded via internal sharing.

> I have the same problem.
> Brave: 1.23.73 Chromium:90.0.4430.93
> I made a post in the community, however, it fell into hold, saying it needs moderation, so far nothing and I can't even comment in the community.

This is a different issue and your best bet is going through https://community.brave.com. CCing @Hollons in case he can help re: why your posts are being flagged as needing moderation.

**Comment by kristoferbartinik** [comment]
I did not understand well, in configurations that release/blocking. Because I did not do any configuration, I just left the default.

**Comment by deeppandya** [comment]
@kjozwiak is this issue relevant ? Should i look into it ?

**Comment by kjozwiak** [comment]
> @kjozwiak is this issue relevant ? Should i look into it ?

Apologies for the delayed response @deeppandya! We can probably close this off as we've stopped publishing `aab` builds under https://github.com/brave/brave-browser/releases and went back to `APK` as we had a lot of issues manually pushing `aab` builds into internal app sharing. Believe we just automated uploading `aab` builds into the internal release channels whenever we do release builds. @srirambv confirmed this was working a few hours ago with https://github.com/brave/brave-browser/releases/tag/v1.27.110.

@SergeyZhukovsky let me know if you disagree re: closing this off.
